dotcom-rendering/src/web/components/EnhancePinnedPost.importable.tsx

```diff
@@ -22,6 +22,18 @@ function hideShowMore() {
 	if (pinnedPostButton) pinnedPostButton.style.display = 'none';
 }
 
+/**
+ * Scroll to the top of the main content when the pinned post is collapsed if the top of the post is out of view
+ */
+function scrollOnCollapse() {
+	const position = pinnedPost?.getBoundingClientRect();
+	if (position && position.top < 0) {
+		pinnedPost?.scrollIntoView({
+			behavior: 'smooth',
```

Comment:
Just a note to remind everyone that this is not supported on Safari–including iOS–, where the scroll will be instantaneous.